src/util/defs.h

```diff
@@ -41,3 +41,4 @@ constexpr Qt::Key kHideRemoveShortcutKey = Qt::Key_Delete;
 #endif
 
 constexpr Qt::Key kRenameSidebarItemShortcutKey = Qt::Key_F2;
+constexpr Qt::Key kRenameSidebarItemAlternativeShortcutKey = Qt::Key_R;
```

Comment:
```suggestion
```
This is not a constexpr it is set by kbd.cfg

Reply:
See [my previous response](https://github.com/mixxxdj/mixxx/pull/13148/files#r1938341361) for why `r` was hardcoded.